pypeit/spectrographs/slitmask.py

```diff
@@ -206,9 +206,9 @@ def __init__(self, corners, slitid=None, align=None, science=None, onsky=None, o
         self.slitindx = None
         if objects is not None:
             self.objects = numpy.atleast_2d(objects)
-            if self.objects.shape[1] != 7:
-                raise ValueError('Must provide the slit ID, sky coordinates, object name, top '
-                                 'and bottom distance for each object.')
+            if self.objects.shape[1] != 9:
+                raise ValueError('Must provide the slit ID, sky coordinates, object name, '
+                                 'object magnitude and band, top and bottom distance for each object.')
```

Comment:
I think the docstring needs to be updated.  It still says `objects` should be a table with 4 columns.

Reply:
Thanks! Done!